tests/common.go

```diff
@@ -7183,22 +7183,25 @@ func EndPxBackupTorpedoTest(contexts []*scheduler.Context) {
 	if ginkgoTestDescr.Failed {
 		log.Infof(">>>> FAILED TEST: %s", ginkgoTestDescr.FullTestText)
 	}
-	log.Infof(">>>> Collecting logs for testcase : %s", ginkgoTestDescr.FullTestText)
-	testCaseName := ginkgoTestDescr.FullTestText
-	matches := regexp.MustCompile(`\{([^}]+)\}`).FindStringSubmatch(ginkgoTestDescr.FullTestText)
-	if len(matches) > 1 {
-		testCaseName = matches[1]
-	}
-	masterNode := node.GetMasterNodes()[0]
-	log.Infof("Creating a directory [%s] to store logs", pxbLogDirPath)
-	err := runCmd(fmt.Sprintf("mkdir -p %v", pxbLogDirPath), masterNode)
-	if err != nil {
-		log.Errorf("Error in creating a directory [%s] to store logs. Err: %v", pxbLogDirPath, err.Error())
-		return
+	masterNodes := node.GetMasterNodes()
+	if masterNodes != nil && len(masterNodes) > 0 {
```

Comment:
`masterNodes != nil` is not required. `len(masterNodes) > 0` will work in all cases

Reply:
Done